omod/src/main/java/org/openmrs/module/owa/activator/OwaActivator.java

```diff
@@ -44,7 +44,7 @@ public void contextRefreshed() {
 		 */
 		String owaAppFolderPath = Context.getAdministrationService().getGlobalProperty(AppManager.KEY_APP_FOLDER_PATH);
 		if (null == owaAppFolderPath) {
-			owaAppFolderPath = OpenmrsUtil.getApplicationDataDirectory() + "owa";
+			owaAppFolderPath = OpenmrsUtil.getApplicationDataDirectory() + "/owa";
```

Comment:
+1 It's also good practice to check if the directory already ends with a file separator

owaAppFolderPath = OpenmrsUtil.getApplicationDataDirectory() +  OpenmrsUtil.getApplicationDataDirectory() .endsWith(File.separator) ? "owa" : File.separator + "owa";